integration_tests/pages/goal/ArchiveGoalPage.ts

```diff
@@ -17,7 +17,7 @@ export default class ArchiveGoalPage extends Page {
   }
 
   enterReason(value: string): ArchiveGoalPage {
-    this.reasonOther().type(value)
+    this.reasonOther().type(value, { delay: 0 })
```

Comment:
All changes in this PR are the same thing - in every place where we call the cypress `.type` method we are now specifying the optional options object with the delay (between each "key press") set to 0